pandas/plotting/_misc.py

```diff
@@ -29,7 +29,7 @@ def table(ax, data, rowLabels=None, colLabels=None, **kwargs):
 
 def register():
     """
-    Register Pandas Formatters and Converters with matplotlib.
+    Register pandas Formatters and Converters with matplotlib.
 
     This function modifies the global ``matplotlib.units.registry``
     dictionary. Pandas adds custom converters for
```

Comment:
```suggestion
    dictionary. pandas adds custom converters for
```

Reply:
Is it not the case of start of a sentence?

Reply:
Our standard, not enforced much so far, is to use `pandas` always lowercase, like a trademark. Not sure where it comes from, but we already have a PR that will validate it's always lowercase in the web and the docs, and we'll follow up with one for the code, so if you can fix it here that would be great (otherwise we'll fix it later).